ext/node/polyfills/process.ts

```diff
@@ -74,7 +78,14 @@ const notImplementedEvents = [
 export const argv = [];
 
 // Overwrites the 1st item with getter.
-Object.defineProperty(argv, "0", { get: Deno.execPath });
+// TODO(bartlomieju): added "configurable: true" to make this work for binary
+// commands, but that is probably a wrong solution
+Object.defineProperty(argv, "0", {
```

Comment:
Feels like we should set up all values for `argv` in `__bootstrapNodeProcess`

Reply:
I'd like to address it in a follow up though. It's low priority

Reply:
add todo comment?